Gems/Pointcloud/Code/Source/Tools/Components/PointcloudEditorComponent.cpp

```diff
@@ -66,19 +66,14 @@ namespace Pointcloud
         AZ::SystemTickBus::QueueFunction(
             [this]()
             {
+                m_scene = AZ::RPI::Scene::GetSceneForEntityId(GetEntityId());
                 if (m_scene && m_pointcloudAsset)
                 {
                     m_featureProcessor = m_scene->EnableFeatureProcessor<PointcloudFeatureProcessor>();
                     AZ_Assert(m_featureProcessor, "Failed to enable PointcloudFeatureProcessorInterface.");
-                    m_pointcloudAsset.QueueLoad();
-                    m_pointcloudAsset.BlockUntilLoadComplete();
-                    m_numPoints = m_pointcloudAsset->m_data.size();
-                    AZStd::vector<AZStd::vector<PointcloudAsset::CloudVertex>> cloudVertexDataChunks;
-
-                    m_pointcloudHandle = m_featureProcessor->AcquirePointcloud(m_pointcloudAsset->m_data);
-
-                    if (m_pointcloudHandle != PointcloudFeatureProcessorInterface::InvalidPointcloudHandle)
+                    if (m_pointcloudAsset)
```

Comment:
visibility should be taken into account during component activation. Currently if the entity is marked as hidden pointcloud is still rendered upon level activation